Add Tracer Documentation (#2017)
chaos-polymtl/lethe: doc/source/theory/multiphysics/passive_tracer/passive_tracer_equations.rst

Passive Tracer Equations
=========================

> Lethe allows the simulation of a passive tracer, which is governed by the equation:

Reviewer: ```suggestion
Lethe allows the simulation of a passive tracer, which is governed by the following advection-diffusion equation:
```

.. math::
    \frac{\partial C}{\partial t} + \mathbf{u} \cdot \nabla C - \kappa \nabla^2 C + f = 0,

where:

* :math:`C` is the tracer concentration;
* :math:`\mathbf{u}` is the velocity of the fluid;
* :math:`\nabla` is the `del operator <https://en.wikipedia.org/wiki/Del>`_;
* :math:`\kappa` is the diffusivity coefficient;
* :math:`f` is the tracer source term per unit of mass.

.. important::

>     The tracer can be advected with the same velocity of the fluid, or a `drift velocity <../../../parameters/cfd/tracer_drift_velocity.html>`_ can be added. A reactive tracer feature also allows the prescription of a tracer consumption rate.

Reviewer 2: ```suggestion
    The tracer is advected by the fluid velocity, to which can be added a `drift velocity <../../../parameters/cfd/tracer_drift_velocity.html>`_ . A reactive tracer feature also allows the prescription of a tracer consumption rate.
```

Reviewer: I like this suggested formulation.

> To obtain the finite element discretization, we start from the strong form previously presented and integrate it over a domain :math:`\Omega` with boundary :math:`\Gamma` such that:

Reviewer 2: ```suggestion
To obtain the finite element discretization, we start from the strong form previously presented, multiply it by a test function then integrate it over the domain :math:`\Omega` with boundary :math:`\Gamma` such that:
```

Reviewer: I would mention that the test function is q so that there is no ambiguity.

Reviewer: You can have a strong form without the test function, so the previous writing also worked. It's just a question of wording.  You need to introduce the test function the moment you perform the integral essentiallyh. So when  you have the strong integral form.

.. math::

>     \int_{\Omega} \left( \frac{\partial C}{\partial t} + \mathbf{u} \cdot \nabla C - \kappa \nabla^2 C - f \right) \: d\Omega = 0.

Reviewer 2: ```suggestion
    \int_{\Omega} q \left( \frac{\partial C}{\partial t} + \mathbf{u} \cdot \nabla C - \kappa \nabla^2 C - f \right) \: d\Omega = 0.
```

> To obtain the corresponding weak form, we multiply the equation above by a test function :math:`q` and apply integration by parts on the Laplacian term:

Reviewer 2: ```suggestion
To obtain the corresponding weak form, we then apply integration by parts on the Laplacian term:
```
